### PR TITLE
Add some docs

### PR DESCRIPTION
**Pull request by a-coding**

### CODE REVIEW

File: docs/blackjack/doc.md [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+Of course the common errors like `Insufficient gas` may be encountered that are particular to Sui.
+
+`DoubleUpClient.createBlackjackGame` 
+
+With a `Table` created this is the next call in order. It will create a `Game` that will be attached to the `Table` and it’s data can be found with the first call explained: `DoubleUpClient.getBlackjackTable`, in the response under `currentGame`.
+
+**Arguments**
+
+The call takes one object as argument that has 3 fields, `coinType`, `coin` and `transaction`.
+The first is a string, the second is a string and it’s the ID of the coin to be used and the last is a `Transaction`. Notice that here a coin is required and it’s value should be within accepted limits, for SUI coins, the lowest limit is 1 and the highest depends on the house, typically it is around 5000.
```
Comment (jonahsimon13) [inline docs/blackjack/doc.md:231]: the type here is TransactionObjectArgument

File: docs/blackjack/doc.md [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+ wallet.signAndExecuteTransaction with the signer automatically provided. */
+ 
+ // The signer here is the user's keypair, typically the wallet has that and you
+ // don't need this.
+ const signer = getSigner();
+ const coinType = "0x2::sui::SUI";
+ const betAmount = 1_000_000_000; // this should be stored when it was first requested for the DoubleUpClient.createBlackjackGame call.
+ 
+ const playerDOUBLEDOWN = async () => {
+    const tx = Transaction();
```
Comment (jonahsimon13) [inline docs/blackjack/doc.md:354]: new Transaction()

File: docs/blackjack/doc.md [hunk 3]
```diff
@@ -0,0 +1,453 @@
...
+    const coin = tx.splitCoins(tx.gas, [tx.pure.u64(betAmount)]);
```
Comment (jonahsimon13) [inline docs/blackjack/doc.md:355]: [coin]

File: docs/blackjack/doc.md [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+ wallet.signAndExecuteTransaction with the signer automatically provided. */
+ 
+ // The signer here is the user's keypair, typically the wallet has that and you
+ // don't need this.
+ const signer = getSigner();
+ const coinType = "0x2::sui::SUI";
+ const userCoinId = "0xaabbccddeeff123...";
+ 
+ const createGame = async () => {
+   const tx = new Transaction();
```
Comment (jonahsimon13) [inline docs/blackjack/doc.md:261]: add a splitCoins call to the tx to input as the coin

**Review by jonahsimon13 — APPROVED** [review]

File: docs/blackjack/doc.md (hunk 3 above)
Comment (a-coding) [reply docs/blackjack/doc.md:355]: Does this work? I think that if you split two coins this won't work:
`const [coin1, coin2] = tx.splitCoins(...)`

**Review by jonahsimon13 — APPROVED** [review]

**Pull request merged**